cla_backend/apps/reports/sql/MIExtractByOutcomeTellUsAboutYourProblem.sql

```diff
@@ -0,0 +1,39 @@
+WITH log_changed_category as (
+    SELECT log_created.case_id, (log_changed_category.patch #>> '{}')::json#>>'{backwards,0,value}' as "diagnosis_category"
+    FROM cla_eventlog_log as log_created
+    JOIN cla_eventlog_log as log_changed_category ON log_created.case_id=log_changed_category.case_id
+        AND log_created.notes = 'Case created digitally' AND log_changed_category.notes LIKE 'Changed category to%%'
+)
+SELECT
+  c.personal_details_id as "Personal Details Id"
+  ,c.reference as "Case Id"
+  ,c.source as "Source"
+  ,to_char(c.created, 'YYYY-MM-DD') as "Created"
+  ,to_char(c.modified, 'YYYY-MM-DD') as "Modified"
```

Comment:
Should these above three fields be exported, as they are not part of the report spec.

Reply:
Removed source as not part of spec; have kept in created and modified as date information is always useful in reports

Reply:
@TawneeOwl @BenMillar-CGI the source field was requested by Rachel after demoing the work to her

Reply:
Thanks @said-moj, I have re-added the source field with commit [6c4dc78](https://github.com/ministryofjustice/cla_backend/pull/1143/commits/6c4dc78008ba44a16cfc300cfd6bc051d7689154).